src/test/java/homework14/SeleniumWebDriver.java

```diff
@@ -0,0 +1,142 @@
+package homework14;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+import java.time.Duration;
+
+import static org.junit.Assert.assertEquals;
+
+public class SeleniumWebDriver {
+    public static final By ELEMENTS_CARD = By.xpath("/html/body/div[2]/div/div/div[2]/div/div[1]/div/div[3]/h5");
```

Comment:
Do not use xpath locators with absolute path `/html/body/div[2]/div/div/div[2]/div/div[1]/div/div[3]/h5`, they will never be stable, and it is impossible to determine the element you are looking for
e.g. in our course we addressed the same element like `//div[@class='card-body']/h5[text()='Elements']`